### PR TITLE
47144_course_edit_update

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

**Comment by mdmahmudurrahman** [comment]
Ok

**Comment by shakil-fg** [comment]
OK

File: app/controllers/supervisor/courses_controller.rb [hunk 1]
```diff
@@ -1,8 +1,21 @@
 class Supervisor::CoursesController < ApplicationController
+  before_action :load_course
```
Comment (dieunb) [inline app/controllers/supervisor/courses_controller.rb]: why don't use cancancan to handle it

Comment (noor-framgia) [inline app/controllers/supervisor/courses_controller.rb]: cancancan is not implemented yet. that is in another task :(

**Pull request merged**